### PR TITLE
Design

### PR DESCRIPTION
**Pull request by garcia50**
Hey guys its still a WIP, but please to give me feedback

### CODE REVIEW

**Review by lnchambers — APPROVED** [review]
It fits in really well with our general website theme of random. One thing I noticed is that your nav bar isn't lined up with the template. Could you get that changed in the final version? It feels very weird when you click between views.

File: app/models/item.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 class Item < ActiveRecord::Base
-  validates_presence_of :title, :description, :price, :image
```
Comment (annaroyer) [inline app/models/item.rb:2]: ?

File: app/public/merchant.css [hunk 2]
```diff
@@ -26,7 +26,7 @@ a:active {
 }
 
 .logo {
-  background-image: url("images/little_shop_rusty50.png");
+  background-image: : url("images/little_shop_rusty50.png");
```
Comment (annaroyer) [inline app/public/merchant.css:29]: ?

**Review by annaroyer — APPROVED** [review]
That looks good!

**Pull request closed without merging**